nodes/widgets/ui_control.js

```diff
@@ -13,8 +13,12 @@ module.exports = function (RED) {
         function updateStore (all, items, msg, prop, value) {
             items.forEach(function (item) {
                 const i = all[item]
-                // update the state store for each page
-                statestore.set(ui, i, msg, prop, value)
+                if (i) {
+                    // update the state store for each page
+                    statestore.set(ui, i, msg, prop, value)
+                } else {
+                    node.error('No item with the name ' + item + ' found')
```

Comment:
```suggestion
                    node.error('No item with the name ' + item + ' found', msg)
```

If `msg` is not present in `node.error`, it cannot be caught by the `catch` node.